examples/policies/fixed-issuance-policy/fixed-issuance-policy-v1.pact

```diff
@@ -0,0 +1,123 @@
+(namespace (read-msg 'ns))
+
+(module fixed-issuance-policy-v1 GOVERNANCE
+
+  @doc "Policy for minting with a fixed issuance"
+
+  (defconst ADMIN-KS:string "marmalade-v2.marmalade-contract-admin")
```

Comment:
We can add 
`marmalade-examples.marmalade-examples-admin` keyset in marmalade.repl (L76~) like the v2 and sale namespaces, and then update ADMIN-KS